qa/tasks/cephfs/test_volumes.py

```diff
@@ -27,14 +27,20 @@ class TestVolumes(CephFSTestCase):
     def _fs_cmd(self, *args):
         return self.mgr_cluster.mon_manager.raw_cluster_cmd("fs", *args)
 
-    def _generate_random_subvolume_name(self):
-        return "{0}_{1}".format(TestVolumes.TEST_SUBVOLUME_PREFIX, random.randint(0, 10000))
+    def _generate_random_subvolume_name(self, count=1):
+        r = random.sample(range(10000), count)
```

Comment:
I think it's okay to reduce the range, but better make these new functions handle errors?

```
2019-11-06 05:05:26,322.322 INFO:__main__:======================================================================
2019-11-06 05:05:26,322.322 INFO:__main__:ERROR: test_subvolume_ls (tasks.cephfs.test_volumes.TestVolumes)
2019-11-06 05:05:26,323.323 INFO:__main__:----------------------------------------------------------------------
2019-11-06 05:05:26,323.323 INFO:__main__:Traceback (most recent call last):
2019-11-06 05:05:26,323.323 INFO:__main__:  File "/home/jcollin/workspace/ceph-review/qa/tasks/cephfs/test_volumes.py", line 473, in test_subvolume_ls
2019-11-06 05:05:26,323.323 INFO:__main__:    subvolumes = self._generate_random_subvolume_name(101)
2019-11-06 05:05:26,323.323 INFO:__main__:  File "/home/jcollin/workspace/ceph-review/qa/tasks/cephfs/test_volumes.py", line 31, in _generate_random_subvolume_name
2019-11-06 05:05:26,323.323 INFO:__main__:    r = random.sample(range(10), count)
2019-11-06 05:05:26,323.323 INFO:__main__:  File "/usr/lib64/python2.7/random.py", line 320, in sample
2019-11-06 05:05:26,323.323 INFO:__main__:    raise ValueError("sample larger than population")
2019-11-06 05:05:26,323.323 INFO:__main__:ValueError: sample larger than population
2019-11-06 05:05:26,323.323 INFO:__main__:
2019-11-06 05:05:26,323.323 INFO:__main__:----------------------------------------------------------------------
2019-11-06 05:05:26,324.324 INFO:__main__:Ran 1 test in 26.944s
2019-11-06 05:05:26,324.324 INFO:__main__:
2019-11-06 05:05:26,324.324 INFO:__main__:FAILED (errors=1)
```

Reply:
`random.sample()` would raise that exception anyway -- that would be caught while testing.